Remove names from the troubleshooting page (#255)
NUbots/NUbook: src/book/03-guides/04-general/03-troubleshooting.mdx

## General

> If this page does not solve your problem, ask for help on Slack either in a channel or directly to a senior team member.

Reviewer: ```suggestion
If this page does not solve your problem, ask for help on Slack either in a channel by directly messaging a senior team member.
```

Author: I think you need to reread what you put :stuck_out_tongue: 
Maybe
```suggestion
If this page does not solve your problem, ask for help on Slack by messaging a channel or messaging a senior team member directly.
```

Reviewer 2: @LachlanCourt  ^^^ This suggestion changes the meaning of the sentence though right? I read the original sentence as "ask in Slack" or "ask a senior member directly". The latter could be in person etc.
